libsolidity/formal/CHC.cpp

```diff
@@ -262,7 +266,8 @@ void CHC::endVisit(FunctionDefinition const& _function)
 	if (
 		!_function.isConstructor() &&
 		_function.isPublic() &&
-		contractFunctions(*m_currentContract).count(&_function)
+		contractFunctions(*m_currentContract).count(&_function) &&
+		m_currentContract->canBeDeployed()
```

Comment:
Not sure about this one. What about inherited functions?

Reply:
In which situation do you think this would not be correct?
(The regression tests did not show any problem.)
Inherited functions in the abstract contract or inherited from the abstract contract?
Maybe I know what you mean, I will try some example.

Reply:
There is `modifiers/modifier_overriding_4.sol` in the regression tests, where assertion violations are still reported in a function of an abstract contract, when they are called in the context of concrete contracts.
Is that the scenario you had in mind?

Reply:
Ok yea, that's what I had in mind, so I guess it's fine.

Reply:
I believe it should be. I'll rebase and add a changelog entry.